Xamarin.Forms.Platform.Android/Renderers/SliderRenderer.cs

```diff
@@ -84,20 +90,20 @@ protected override void OnLayout(bool changed, int l, int t, int r, int b)
 			base.OnLayout(changed, l, t, r, b);
 
 			BuildVersionCodes androidVersion = Build.VERSION.SdkInt;
-			if (androidVersion >= BuildVersionCodes.JellyBean)
-			{
-				// Thumb only supported JellyBean and higher
+			if (androidVersion < BuildVersionCodes.JellyBean)
+				return;
 
-				if (Control == null)
-					return;
+			// Thumb only supported JellyBean and higher
 
-				SeekBar seekbar = Control;
+			if (Control == null)
+				return;
 
-				Drawable thumb = seekbar.Thumb;
-				int thumbTop = seekbar.Height / 2 - thumb.IntrinsicHeight / 2;
+			SeekBar seekbar = Control;
 
-				thumb.SetBounds(thumb.Bounds.Left, thumbTop, thumb.Bounds.Left + thumb.IntrinsicWidth, thumbTop + thumb.IntrinsicHeight);
-			}
+			Drawable thumb = seekbar.Thumb;
+			int thumbTop = seekbar.Height / 2 - thumb.IntrinsicHeight / 2;
+
+			thumb.SetBounds(thumb.Bounds.Left, thumbTop, thumb.Bounds.Left + thumb.IntrinsicWidth, thumbTop + thumb.IntrinsicHeight);
```

Comment:
except for reducing nesting (I like that), have you changed something in this block ?